src/components/Dialogs/utils.ts

```diff
@@ -8,8 +8,17 @@ export interface MetricModalContent {
   howItsCalculated: string;
   dataSource?: React.ReactElement;
   metricDefinition: string;
+  learnLink: string;
 }
 
+const metricToLearnLink: { [key in Metric]: string } = {
+  [Metric.CASE_DENSITY]: '/covid-risk-levels-metrics#daily-new-cases',
+  [Metric.CASE_GROWTH_RATE]: '/covid-risk-levels-metrics#infection-rate',
+  [Metric.HOSPITAL_USAGE]: '/covid-risk-levels-metrics#icu-capacity-used',
+  [Metric.POSITIVE_TESTS]: '/covid-risk-levels-metrics#positive-test-rate',
+  [Metric.VACCINATIONS]: '/covid-risk-levels-metrics#percent-vaccinated',
+};
```

Comment:
Can we grab this from the CMS? `src/cms-content/tooltips/metric-calculation-tooltips.json` has the CTA (even if we are not reusing the title)

Reply:
and by down the line i mean.. tomorrow or thursday